File: Beginner/1066 (Even, Odd, Positive and Negative)/1066.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            int acmPar = 0;
            int acmImpar = 0;
            int acmPositivo = 0;
            int acmNegativo = 0;
           
            for(int i = 0; i < 5; i++)
            {
                int x = int.Parse(Console.ReadLine());
                if (x > 0)
                {
                    acmPositivo++;
                }
                else if(x < 0)
                {
                    acmNegativo++;
                }
                if(x % 2 == 0)
                {
                    acmPar++;
                }
                else
                {
                    acmImpar++;
                }
            }

            Console.WriteLine(acmPar + " valor(es) par(es)");
            Console.WriteLine(acmImpar + " valor(es) impar(es)");
            Console.WriteLine(acmPositivo + " valor(es) positivo(s)");
            Console.WriteLine(acmNegativo + " valor(es) negativo(s)");
        }
    }
}

File: Beginner/1066 (Even, Odd, Positive and Negative)/1066.py
acmPar = int() 
acmImpar = int()
acmPositivo = int()
acmNegativo = int()

for i in range (5):
    x = int(input())
    if x > 0:
        acmPositivo += 1
    elif x < 0:
        acmNegativo +=1
    if (x % 2) == 0:
        acmPar += 1
    else:
        acmImpar +=1
print(acmPar, "valor(es) par(es)")
print(acmImpar, "valor(es) impar(es)")
print(acmPositivo, "valor(es) positivo(s)")
print(acmNegativo, "valor(es) negativo(s)")
  
